R/RobotZoo/Package.toml

```diff
@@ -1,3 +1,3 @@
 name = "RobotZoo"
 uuid = "74be38bb-dcc2-4b9e-baf3-d6373cd95f10"
-repo = "https://github.com/bjack205/RobotZoo.jl.git"
+repo = "https://github.com/RobotExplorationLab/RobotZoo.jl.git"
```

Comment:
Seems like a typo. Should be `Robotic`?

Reply:
Bump @bjack205 

Seems like the new URL should be `https://github.com/RoboticExplorationLab/RobotZoo.jl`

Reply:
ah, yes, sorry about that. It's now fixed.